tuf/client/updater.py

```diff
@@ -1561,85 +1659,73 @@ def _update_metadata_if_changed(self, metadata_role,
 
 
 
-  def _fileinfo_has_changed(self, metadata_filename, new_fileinfo):
+  def _versioninfo_has_changed(self, metadata_filename, new_versioninfo):
```

Comment:
Resolved in PR #321, see [above](https://github.com/theupdateframework/tuf/pull/295#discussion_r54261363).